music21/clef.py

```diff
@@ -1085,6 +1097,26 @@ def testContexts(self):
         self.assertIs(n5.getContextByClass(Clef), tc1)
         self.assertIs(n6.getContextByClass(Clef), tc1)
 
+    def testTabClefBeamDirections(self):
+
+        from music21 import stream
+        from music21 import clef
+        from music21 import meter
+        from music21 import note
+
+        m = stream.Measure()
+
+        n1 = note.Note(64, quarterLength=0.25)
+        n2 = note.Note(67, quarterLength=0.25)
+
+        m.append(clef.TabClef())
+        m.append(meter.TimeSignature('4/4'))
+        m.append(n1)
+        m.append(n2)
+        m = m.makeBeams()
```

Comment:
Very minor detail, but I would just use `inPlace=True` rather than overwriting `m`.

Reply:
Done. Didn't notice this `inPlace` parameter, thanks.